distribution/registry.go

```diff
@@ -71,14 +71,11 @@ func NewV2Repository(
 		DualStack: true,
 	}
 
-	// TODO(dmcgowan): Call close idle connections when complete, use keep alive
 	base := &http.Transport{
 		Proxy:               http.ProxyFromEnvironment,
 		DialContext:         direct.DialContext,
 		TLSHandshakeTimeout: 10 * time.Second,
 		TLSClientConfig:     endpoint.TLSConfig,
-		// TODO(dmcgowan): Call close idle connections when complete and use keep alive
```

Comment:
Looks like this was disabled because closing idle connections was not yet implemented; is that done now?

@dmcgowan @yedamao ?

Reply:
```
// By default, Transport caches connections for future re-use.
// This may leave many open connections when accessing many hosts.
// This behavior can be managed using Transport's CloseIdleConnections method
// and the MaxIdleConnsPerHost and DisableKeepAlives fields.
//
```
I think the DefaultMaxIdleConnsPerHost can avoid leave many open connections. Not so sure  🤣

Reply:
Yeah, just making sure we're not introducing a problem with this change 😅 

I'll wait for @dmcgowan to have a look 👍

Reply:
What are these values set to though? Has this changed in Go since this was originally added, originally we had to add this because of memory leaks. In general, after a pull/push operations there should be no remaining connections open to the host, would that still be the case here?